utils/jasutils/jasutils.go

```diff
@@ -82,6 +86,22 @@ var applicableMapToScore = map[string]int{
 	"NotApplicable":             0,
 }
 
+var TokenValidationOrder = map[string]int{
+	"Active":      1,
+	"Inactive":    2,
+	"Unsupported": 3,
+	"Unavailable": 4,
+	"":            5,
+	"Not a token": 6,
+}
```

Comment:
Create an Enum for the keys (SecretStatus), use it instead of Const

Reply:
done